packages/pds-ember/.storybook/main.js

```diff
@@ -2,6 +2,12 @@ module.exports = {
   addons: [
     '@storybook/addon-docs',
     '@storybook/addon-knobs',
+    {
+      name: '@storybook/addon-essentials',
+      options: {
+        backgrounds: false,
```

Comment:
Backgrounds disabled for now. It didn't seem to be working at the time. We'll revisit its necessity, in the future.

Reply:
is `addon-essentials` a new core addon for SB6?

Reply:
looks like yes (https://storybook.js.org/docs/react/essentials/introduction) - does it conflict with knobs and docs? do those need to be removed?

Reply:
Judging from the `yarn.lock` changes, I think you can remove `addon-docs` (included in `addon-essentials`) and `addon-knobs` (replaced by `addon-controls`).

Reply:
you can have knobs and controls side-by-side, but a story will only ever use one of them...

For example: _(Components > Button > Index)_ Knobs is still used, but Controls is present with a banner.
![CleanShot 2020-10-16 at 13 30 24](https://user-images.githubusercontent.com/545605/96295751-c2ff3580-0fb3-11eb-958b-eba0af3332bb.png)

This kind of thing will be common as we transition stories to SB6 format.